CONP_DATS_fields.md

```diff
@@ -1,51 +1,52 @@
-|primaryPublications|**RECOMMENDED**. The primary publication(s) associated with the dataset, usually describing how the dataset was produced.|
-|dimensions|**RECOMMENDED**. The different dimensions (granular components) making up a dataset. Providing dimensions give more details about the data types.|
-|identifier|**RECOMMENDED**. Primary identifier for the dataset. Provide a *Document Object Identifier (DOI)* if you have one.|
-|extraProperties - contact|**RECOMMENDED**. Provide contact information (name, email, telephone) of the person responsible for the dataset.|
-|extraProperties - logo|**RECOMMENDED**. Link to or provide a logo to display on your resource landing page.|
-|creators - name - roles - value("Principal Investigator")|**RECOMMENDED** Indicate which of the creators is PI for this dataset.  May not be applicable in cases where the value of "creators" is an organisation rather than a person or list of people.|
-|dates|**OPTIONAL**.  Relevant dates for the dataset. If you provide a date, it must come with a description of the date.|<!--will later choose from a pulldown list--> 
-|citations|**OPTIONAL**.  Publication(s) citing this dataset.|
-|citationCount|**OPTIONAL**. The number of publications that cite this dataset (enumerated in the citations property).|
-|producedBy|**OPTIONAL**. Process which generated a given dataset.|
-|isAbout|**OPTIONAL**. Entities (biological entity, taxonomic information, disease, molecular entity, anatomical part, treatment) associated with this dataset.|
-|hasPart|**OPTIONAL**. A dataset that is a subset of this dataset; datasets declaring the 'hasPart' relationship are considered a collection of datasets.  The aggregation criteria should be included in the 'description' field.|
-|acknowledges|**OPTIONAL**. Grant(s) which funded and supported the work reported by the dataset.|
-|refinement|**OPTIONAL**. Qualifier describing the level of data processing of the dataset and its distributions.|
-|aggregation|**OPTIONAL**. Qualifier indicating whether the entity represents an 'instance of a dataset' or a 'collection of datasets'.|
-|spatialCoverage|**OPTIONAL**. The geographical extension and span covered by the dataset and its measured dimensions/variables.|
+|Portal field name|DATS schema field|Status|Description|Portal help text|
+|-|-|-|-|-|
+|Title|title|**REQUIRED**| Name of the dataset. | Name of the dataset. The title needs to be short and easily recognizable and searchable. If there is an acronym, please also provide the long name.|
+|Creators|creators|**REQUIRED**| The person(s) or organization(s) which contributed to the creation of the dataset. | The person(s) or organization(s) which contributed to the creation of the dataset. This can be the principal investigator, hospital, university, centre, clinic, etc. The first contact provided will be associated as the main contact for this dataset.|
```

Comment:
I would say "that contributed" rather than "which contributed" in this line, can someone confirm whether that is correct North American English usage ?